tests/django_restframework_gis_tests/tests.py

```diff
@@ -472,6 +478,15 @@ def test_post_geojson_location_list_HTML_web_form_WKT(self):
         location = Location.objects.all()[0]
         self.assertEqual(location.name, "HTML test WKT")
 
+    @skipIf(is_pre_drf_39, 'Skip this test if DRF < 3.9')
+    def test_geojson_HTML_widget_value(self):
```

Comment:
I'm not sure if this is correct. This test method shares the same name as below, so it should be overwritten by the latter, right?